includes/data-stores/class-wc-product-data-store-cpt.php

```diff
@@ -1440,7 +1440,7 @@ public function search_products( $term, $type = '', $include_variations = false,
 			$product_ids[] = wp_get_post_parent_id( $post_id );
 		}
 
-		return wp_parse_id_list( $product_ids );
+		return apply_filters( 'woocommerce_search_products_ids', wp_parse_id_list( $product_ids ) ) );
```

Comment:
This has no context, how about including some of the args passed to `search_products` also? `$term, $type = '', $include_variations....`